XamlControlsGallery/ControlPages/InkToolbarPage.xaml.cs

```diff
@@ -26,9 +26,12 @@ private void UpdatePen()
             {
                 var defaultAttributes = _inkPresenter.CopyDefaultDrawingAttributes();
 
-                defaultAttributes.PenTip = (bool)penTipShape.IsChecked ? PenTipShape.Circle : PenTipShape.Rectangle;
-
-                _inkPresenter.UpdateDefaultDrawingAttributes(defaultAttributes);
+                // If we are using a pencil, changing pentip is not allowed!
+                if(defaultAttributes.Kind != InkDrawingAttributesKind.Pencil)
```

Comment:
Should this instead be:
```suggestion
                if(defaultAttributes.Kind == InkDrawingAttributesKind.Default)
```

?

I think the enum only has two types so these are effectively the same, but the philosophical difference is about specificity. What is here says "Rectangle should work on everything but Pencil", whereas the above suggestion says "The only place Rectangle is guaranteed to work is on Default".

Reply:
Oh yes, that makes more sense! Great catch!